amical/_cli/commands/clean.py

```diff
@@ -0,0 +1,114 @@
+import os
+from datetime import datetime
+from glob import glob
+from pathlib import Path
+
+from astropy.io import fits
+from matplotlib import pyplot as plt
+from tabulate import tabulate
+from termcolor import cprint
+from tqdm import tqdm
+
+import amical
+
+
+def _select_data_file(args, process):
```

Comment:
still shouldn't pass `args` directly. I won't repeat this comment any more, please check the rest of the function thoroughly